core/rint/CMakeLists.txt

```diff
@@ -22,4 +22,6 @@ ROOT_STANDARD_LIBRARY_PACKAGE(Rint
     Core
 )
 
+target_include_directories(Core PRIVATE inc)
```

Comment:
Does that meant that TRint (and TTabCom) are no longer part of the public interface/installed?

Reply:
I can check that, but `TRint` itself has that in its public interface, it's only private to libCore.